### PR TITLE
Added logging for every uuid for every client

### PR DESCRIPTION
**Pull request by kovacshuni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,26 +16,25 @@ go test ./...
...
-```
-            ./notifications-push \
-                --notifications_resourse="content"
-                --consumer_proxy_addr="https://kafka-proxy-iw-uk-p-1.glb.ft.com,https://kafka-proxy-iw-uk-p-2.glb.ft.com" \
-                --consumer_group_id="notifications-push" \
-                --consumer_authorization_key "$(ssh semantic-tunnel-up.ft.com etcdctl get /ft/_credentials/kafka-bridge/authorization_key)" \
-                --api-base-url="http://api.ft.com" \
-                --topic="CmsPublicationEvents"
-```
+* via environment variables:
```
Comment (tamas-molnar) [inline README.md]: formatting, see https://github.com/Financial-Times/notifications-push/blob/investigate-outage/README.md

File: dispatcher/dispatcher.go [hunk 2]
```diff
@@ -67,12 +67,14 @@ func (d *dispatcher) Start() {
...
 	d.lock.RLock()
 	defer d.lock.RUnlock()
-
 	for sub := range d.subscribers {
+		log.WithField("transaction_id", notification.PublishReference).
+			WithField("resource", notification.APIURL).
+			WithField("subscriberAddress", sub.Address()).
+			WithField("subscriberSince", sub.Since().Format(time.RFC3339)).
+			Info("Forwarding to subscribers.")
 		sub.send(notification)
```
Comment (tamas-molnar) [inline dispatcher/dispatcher.go]: Can this call fail?

File: README.md (hunk 1 above)
Comment (kovacshuni) [inline README.md]: updated.

File: dispatcher/dispatcher.go (hunk 2 above)
Comment (kovacshuni) [inline dispatcher/dispatcher.go]: Yes, it can, i've added logging for that case.

**Pull request merged**